src/scripts/main.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  if (e.target.tagName === 'BODY') {
+    return;
+  }
+
+  const wallElement = document.querySelector('.wall');
+  const spiderElement = document.querySelector('.spider');
+  const wallX = wallElement.getBoundingClientRect().x;
+  const wallY = wallElement.getBoundingClientRect().y;
+  const cursorX = e.clientX;
+  const cursorY = e.clientY;
+  const wallReduceSpider
+    = parseInt(window.getComputedStyle(wallElement).width)
+    - parseInt(window.getComputedStyle(spiderElement).width);
+  const borderAndCenterSpider
+    = parseInt(window.getComputedStyle(spiderElement).width) / 2
+    + parseInt(window.getComputedStyle(wallElement).borderWidth);
```

Comment:
I would add some space
```suggestion
  const wallElement = document.querySelector('.wall');
  const spiderElement = document.querySelector('.spider');
  
  const wallX = wallElement.getBoundingClientRect().x;
  const wallY = wallElement.getBoundingClientRect().y;
  
  const cursorX = e.clientX;
  const cursorY = e.clientY;
  
  const wallReduceSpider
    = parseInt(window.getComputedStyle(wallElement).width)
    - parseInt(window.getComputedStyle(spiderElement).width);
    
  const borderAndCenterSpider
    = parseInt(window.getComputedStyle(spiderElement).width) / 2
    + parseInt(window.getComputedStyle(wallElement).borderWidth);
```